selfify/sidebar/sidebar.css

```diff
@@ -0,0 +1,6 @@
+
+.drop_zone {
+  border: 5px solid blue;
+  width:  400px;
+  height: 200px;
```

Comment:
It would also be good to have some margin for the input element, it's crammed right underneath the title now.